app/components/feedback_link/view.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if enabled %>
+<div class="govuk-inset-text">
+  <h2 class="govuk-heading-s">How are you finding this process?</h2>
+  <p class="govuk-body">
+    <a href=<%= feedback_link_url %>
```

Comment:
Might this use the `govuk_link_to` helper?

Reply:
Good spot, pushed up a change for this.